front/app/containers/HomePage/SignedInHeader/FallbackStep.tsx

```diff
@@ -70,7 +70,7 @@ const FallbackStep = ({
       >
         <HeaderContent id="e2e-signed-in-header-default-cta">
           <Left>
-            <Title variant="h2" m="0" color="white">
+            <Title variant="h2" m="0" color="white" as="h1">
```

Comment:
Unrelated a11y fix made by Iva (not sure where this is coming from, but I'll leave it as it seems to make sense).